File: 백준/Bronze/1159. 농구 경기/농구 경기.cc
#include <bits/stdc++.h>
using namespace std;

int arr[26];

int main()
{
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	

	int n;
	cin >> n;
	string str;

	string ret = "";
	for (int i = 0; i < n; i++){
		cin >> str;
		arr[str[0] -'a']++;
	}

	for (int i =0; i<26; i++){
		if ( 5 <= arr[i] ){
			ret += (i+'a');	
		}
	}

	if(ret.size()){
		cout << ret;
	}else{
		cout << "PREDAJA";
	}

	return 0;
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int N = Integer.parseInt(st.nextToken());

        int[] arr = new int[26];

        String ret = "";

        for (int i = 0; i < N; i++) {
            arr[br.readLine().charAt(0) - 'a']++;
        }

        for (int i = 0; i < 26; i++) {
            if (arr[i] >= 5) {
                ret += (char) (i + 'a');
            }
        }

        if (ret.length() != 0) {
            System.out.println(ret);
        } else {
            System.out.println("PREDAJA");
        }

    }
}